File: 22092021/Super.java
class Super {
	int x = 20;
	
	void show() {
		System.out.println("Hello from Super class");
	}
	
	public static void main(String args[]){
		Sub sb = new Sub();
		System.out.println(sb.x);
		sb.show();
	}
}

class Sub {
	
}

class Class3 extends Super, Sub {
	
}

File: 22092021/Super.cpp
#include <iostream>
using namespace std;

class Super
{
public :
	int x = 20;
	
	void sup_show(){
		cout<<"Hello from show of super"<<endl;
	}
};

class Sub:public Super
{
	
};

int main()
{
	Sub obj;
	cout << obj.x << endl;
	obj.sup_show();
}
